tests/e2e/testfiles/storage_csm_powerscale.yaml

```diff
@@ -11,16 +11,16 @@ spec:
       # Allowed values: ReadWriteOnceWithFSType, File , None
       # Default value: ReadWriteOnceWithFSType
       fSGroupPolicy: "ReadWriteOnceWithFSType"
-    # Config version for CSI PowerScale v2.5.0 driver
-    configVersion: v2.5.0
+    # Config version for CSI PowerScale v2.6.0 driver
+    configVersion: v2.6.0
```

Comment:
Please change this to 2.5 as e2e tests will fail since 2.6 images are not yet posted